Makefile

```diff
@@ -84,7 +84,7 @@ clean:
 	rm -f $(install_dir)/firrtl.jar
 	rm -f $(install_dir)/firrtl
 	rm -f $(install_dir)/firrtl-stanza
-	"$(sbt)" "clean"
+	$(SBT) "clean"
```

Comment:
@yunsup, you may want to include the quotes around $(SBT) so it will work with paths with spaces. However, that would prevent you from including some leading sbt options as part of the $(SBT) definition.